.gitignore

```diff
@@ -16,3 +16,5 @@ yarn-error.log
 /.vscode
 composer.lock
 docker-compose.yml
```

Comment:
This should not be necessary as it is no longer a Laravel app.

Reply:
Oh, they must have been leftover files from a previous copy of the fork.  Apologies.  Will revert those specific changes.